samples/host_verify/README.md

```diff
@@ -0,0 +1,68 @@
+
+# The host verification sample
+
+- Demonstrates the attestation of a remote host
+- Requires only the `openenclave/host_verify.h` header, as well as some C standard header files
+- Also runs in simulation mode and when both BUILD_ENCLAVES and HAS_QUOTE_PROVIDER flags are turned off
+
+Prerequisites: you may want to read [Common Sample Information](../README.md#common-sample-information) before going further
+
+## About the host verification sample
+
+When a host is doing host verification, or remote host attestation to be exact, it means that it is trying to authenticate its hardware and software settings so that it can determine whether or not to trust the remote host.
+This sample does remote host attestation by taking either of the two arguments:
+
+- An SGX report
+- An SGX certificate
+
+A user can generate a report or certificate with `oecert`. See [here](https://github.com/openenclave/openenclave/blob/master/tests/tools/oecert/README.md) for more details.
+
+In the main function, if it sees the atrribute `-r`, then it will continue to read the next argument to get the report and call `verify_report()` to verify the report.
```

Comment:
```suggestion
In the main function, if it sees the attribute `-r`, then it will continue to read the next argument to get the report and call `verify_report()` to verify the report.
```

Reply:
Oops, thanks for catching the typo!